### PR TITLE
update release notes

### PR DESCRIPTION
**Pull request by LeilaGhaffari**

### CODE REVIEW

File: doc/sphinx/source/releasenotes.rst [hunk 1]
```diff
@@ -31,6 +31,8 @@ Examples
 ^^^^^^^^
 * :ref:`example-petsc-elasticity` example updated with traction boundary conditions and improved Dirichlet boundary conditions.
 * :ref:`example-petsc-elasticity` example updated with Neo-Hookean hyperelasticity in current configuration as well as improved Neo-Hookean hyperelasticity exploring storage vs computation tradeoffs.
+* :ref:`example-petsc-navier-stokes` example updated with Traveling Vortex test case which is the analytical solution to the Euler equations.
+* :ref:`example-petsc-navier-stokes` example updated with a support for performing convergence study and plotting the obtained results.
```
Comment (jedbrown) [inline doc/sphinx/source/releasenotes.rst:35]:
```suggestion
* :ref:`example-petsc-navier-stokes` example updated with support for performing convergence study and plotting order of convergence by polynomial degree.
```

File: doc/sphinx/source/releasenotes.rst [hunk 2]
```diff
@@ -31,6 +31,8 @@ Examples
 ^^^^^^^^
 * :ref:`example-petsc-elasticity` example updated with traction boundary conditions and improved Dirichlet boundary conditions.
 * :ref:`example-petsc-elasticity` example updated with Neo-Hookean hyperelasticity in current configuration as well as improved Neo-Hookean hyperelasticity exploring storage vs computation tradeoffs.
+* :ref:`example-petsc-navier-stokes` example updated with Traveling Vortex test case which is the analytical solution to the Euler equations.
```
Comment (jedbrown) [inline doc/sphinx/source/releasenotes.rst:34]:
```suggestion
* :ref:`example-petsc-navier-stokes` example updated with isentropic traveling vortex test case, an analytical solution to the Euler equations that is useful for testing boundary conditions, discretization stability, and order of accuracy.
```

**Review by jedbrown — APPROVED** [review]
Great!
